compiler/z/codegen/OMRMemoryReference.cpp

```diff
@@ -3696,6 +3696,10 @@ generateS390MemoryReference(TR::Register * br, int32_t disp, TR::CodeGenerator *
 TR::MemoryReference *
 generateS390MemoryReference(TR::Node * node, TR::CodeGenerator * cg, bool canUseRX)
    {
+   // The TR::MemoryReference(TR::Node*, TR::CodeGenerator*, bool); constructor (below) is made for store or load nodes
+   // that have symbol references. If the node has no sym ref, this memory reference's symbolRef is null, whichs can lead
```

Comment:
Typo on "whichs". Also can you describe what kind of problems can be encountered in the binary encoding phase? i.e. who will need this symbol reference.

Reply:
Fixed.